test/libsolidity/SolidityABIJSON.cpp

```diff
@@ -495,6 +495,23 @@ BOOST_AUTO_TEST_CASE(empty_name_return_parameter)
 	checkInterface(sourceCode, interface);
 }
 
+BOOST_AUTO_TEST_CASE(constructor_abi)
+{
+	char const* sourceCode = R"(
+		contract test {
+			function test() {
+		}
+	})";
+
+	char const* interface = R"([
+	{
+	   "inputs" : [],
```

Comment:
Could you change the test so that we also get types here? Perhaps even a conversion from a contract type to address?